src/cli/index.js

```diff
@@ -1,5 +1,5 @@
-export * from "./help"
-export * from "./list"
-export * from "./version"
-export * from "./spawn"
-export * from "./options"
+export {help} from "./help"
```

Comment:
I was too excited about modules that I left myself carried away with this stuff. :+1: